Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/self_intersections.h

```diff
@@ -269,7 +269,7 @@ self_intersections(const TriangleMesh& tmesh
 #ifdef DOXYGEN_RUNNING
                  , const NamedParameters& np)
 #else
-                 , const pmp_bgl_named_params<P,T,R>& np)
+                 , const cgal_bgl_named_params<P,T,R>& np)
```

Comment:
The macro `CGAL_PMP_NP_TEMPLATE_PARAMETERS` can be used instead of `class P, class T, class R`
and `CGAL_PMP_NP_CLASS` can be used instead of `cgal_bgl_named_params<P,T,R>`.